course_content/notebooks/numpy_intro.ipynb

```diff
@@ -303,9 +291,7 @@
     "For each of these properties:\n",
     "\n",
     "* Use information from the array documentation to find out more about each array property.\n",
-    "* Apply each property to the array `arr` defined above. Can you explain the results you get in each case?\n",
-    "\n",
-    "Be ready to feed your results back to the group!"
```

Comment:
:open_mouth: :cake: :fork_and_knife: ?

Reply:
The courses are supposed to be useable both as taught training courses in a room and for a person to be able to work through on their own. This particular comment does not fit the second use-case!

But they can all still eat cake.